python/llm/src/ipex_llm/optimize.py

```diff
@@ -170,9 +170,10 @@ def load_low_bit(model, model_path):
         invalidInputError(isinstance(model, torch.nn.Module),
                           "model should be an instance of `torch.nn.Module`, "
                           f"but got {type(model)} at last.")
-        invalidInputError(model.device.type in ('cpu', 'meta'),
```

Comment:
Then how do you ensure that model is on cpu and meta? how about check the status of model.parameters()?